suite-native/device/src/screens/DeviceInfoModalScreen.tsx

```diff
@@ -39,15 +39,18 @@ import { useOpenLink } from '@suite-native/link';
 import { HowToUpdateBottomSheet } from '../components/HowToUpdateBottomSheet';
 
 const deviceImageMap = {
-    [DeviceModelInternal.T1B1]: require('../assets/t1.png'),
-    [DeviceModelInternal.T2T1]: require('../assets/tt.png'),
-    [DeviceModelInternal.T2B1]: require('../assets/ts3.png'),
+    [DeviceModelInternal.T1B1]: require('../assets/t1b1.png'),
+    [DeviceModelInternal.T2T1]: require('../assets/t2t1.png'),
+    [DeviceModelInternal.T2B1]: require('../assets/t2b1.png'),
```

Comment:
this also can be just string interpolation `require('../assets/${device.toLowerCase()}.png'),`

Reply:
Afaik you can't do that in require in React Native

Reply:
yup it is not possible